src/shared/ui/Slider/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import './Slider.scss';
+
+import React, { PropsWithChildren, ReactNode } from 'react';
+import { Autoplay, Navigation, Pagination } from 'swiper';
+import { Swiper, SwiperProps, SwiperSlide } from 'swiper/react';
+
+type Props = {
+  delay?: number;
+} & SwiperProps;
+
+function Slider({ delay = 2500, children, ...props }: PropsWithChildren<Props>) {
+  return (
+    <Swiper
+      modules={[Navigation, Pagination, Autoplay]}
+      slidesPerView={1}
+      slidesPerGroup={1}
+      spaceBetween={30}
+      navigation={true}
+      loop={true}
```

Comment:
No need to pass true explicitly

https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true